client/components/date-range/index.js

```diff
@@ -32,33 +30,6 @@ import './style.scss';
 const NO_DATE_SELECTED_VALUE = null;
 
 export class DateRange extends Component {
-	static propTypes = {
```

Comment:
What's the reason for removing the `propTypes`?

Reply:
Dropping the moment dependency (because of the `instanceof( moment )`). Alternatively, I could turn this into the less strict `object`. How does that sound?

Reply:
Ok, do all the `moment` instances become `Date`s?

Let's keep what we can, at least until we can migrate to TypeScript 😎

Reply:
> Ok, do all the `moment` instances become `Date`s?

No, the PR doesn't start untangling `moment` just yet; it's only getting rid of `i18n-calypso`'s moment.

Alright, I'll reinstate the `propTypes` using an `import 'moment'` instead 👍